src/metal.cs

```diff
@@ -293,6 +314,16 @@ partial interface MTLCommandBuffer {
 		[iOS (10,3)][TV (10,2)][Mac (10,12,4, onlyOn64 : true)]
 		[Export ("GPUEndTime")]
 		double /* CFTimeInterval */ GpuEndTime { get; }
+
+		[Mac (10,13, onlyOn64: true), iOS (11,0), TV (11,0), NoWatch]
+		[Abstract]
```

Comment:
We can't add `[Abstract]` to new members on existing protocols 😒 

The `[Abstract]` line have to be inside `#if XAMCORE_4_0`.